projects/api-client/src/api.module.ts

```diff
@@ -11,6 +11,7 @@ import { HealthService } from './api/health.service';
 import { MetaService } from './api/meta.service';
 import { ProjectService } from './api/project.service';
 import { ProviderService } from './api/provider.service';
+import { TestResultService } from './api/testResult.service';
```

Comment:
These projects regen seems to still be bloating the PR. As #91 has now been merged, could you do another merge or rebase from master?